Library/ViewModels/DashboardViewModelTests.swift

```diff
@@ -557,10 +557,16 @@ internal final class DashboardViewModelTests: TestCase {
         titleViewDataClosed1, titleViewDataOpen1, titleViewDataClosed1,
         titleViewDataOpen1, titleViewDataClosed2, titleViewDataOpen2
       ], "Update title with open data")
-      self.presentProjectsDrawer.assertValues([
+
+      let presentProjectDrawerProjectIds = self.presentProjectsDrawer.values
+        .flatMap { $0.map { $0.project.id } }
+      let projectDataIds = [
         [projectData1, projectData2], [projectData1, projectData2],
         [projectData1, projectData2]
-      ])
+      ]
+      .flatMap { $0.map { $0.project.id } }
+      XCTAssertEqual(presentProjectDrawerProjectIds, projectDataIds)
```

Comment:
Perhaps not the neatest but this keeps the same logic that the tests had before by only comparing on `project.id`.